.github/pull_request_template.md

```diff
@@ -1,5 +1,4 @@
-### Description
-<!--- Describe your changes in detail -->
+This PR makes the following changes:
```

Comment:
This is still going to show up *below* your auto-filled commit description

Reply:
sigh, fine, it was mostly just for testing anyway.